Zend/tests/approx_equals.phpt

```diff
@@ -0,0 +1,52 @@
+--TEST--
+approx equality
+--FILE--
+<?php
+
+// Number compares
+var_dump(2 ~= 1); // false
+var_dump(1.4 ~= 1); // true
+var_dump(-1.4 ~= -1); // true
+var_dump(-1.5 ~= -1.8); // true
+var_dump(random_int(1, 1) ~= 1.1); // true
```

Comment:
Please add a test for the common floating point example to make sure it is correctly handled:

```suggestion
var_dump(random_int(1, 1) ~= 1.1); // true
var_dump(0.1 + 0.2 ~= 0.3); // true
```